lib/stream.js

```diff
@@ -193,7 +193,7 @@ export const getSpawnedResult = async ({
 	} catch (error) {
 		spawned.kill();
```

Comment:
This is a noop when `exitPromise` has resolved.

When either the `exit` or `error` event is emitted, `childProcess._handle` is set to `null`.

https://github.com/nodejs/node/blob/8a41d9b636be86350cd32847c3f89d327c4f6ff7/lib/internal/child_process.js#L282
https://github.com/nodejs/node/blob/8a41d9b636be86350cd32847c3f89d327c4f6ff7/lib/internal/child_process.js#L420

`.kill()` is a noop when `childProcess._handle` is `null`.

https://github.com/nodejs/node/blob/8a41d9b636be86350cd32847c3f89d327c4f6ff7/lib/internal/child_process.js#L498